src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -4,25 +4,31 @@
             [puppetlabs.trapperkeeper.core :refer [defservice]]
             [puppetlabs.services.master.master-core :as core]
             [puppetlabs.puppetserver.certificate-authority :as ca]
-            [puppetlabs.trapperkeeper.services :as tk-services]))
+            [puppetlabs.trapperkeeper.services :as tk-services]
+            [puppetlabs.dujour.version-check :as version-check]))
 
 (defservice master-service
   [[:WebroutingService add-ring-handler get-route]
    [:PuppetServerConfigService get-config]
    [:RequestHandlerService handle-request]
    [:CaService initialize-master-ssl! retrieve-ca-cert! retrieve-ca-crl!]
-   [:JRubyPuppetService]]
+   [:JRubyPuppetService]
+   [:ConfigService get-in-config]]
```

Comment:
This already depends on the `PuppetServerConfigService`, can we just get the config from that?  The entire `config` is pulled in here: https://github.com/puppetlabs/puppet-server/pull/409/files#diff-f69c98ef06ea1897ed3b68e38fb638d3R21

(Honest question - I don't remember if you actually can use the `PuppetServerConfigService` for this)